utils/csv_to_nc/README.md

```diff
@@ -8,4 +8,4 @@ convention table](https://docs.google.com/spreadsheets/d/1CA3aTvI9piXqRqO-3MGrsH
 
 The MATLAB source code writes model outputs in `csv` format in the output
 directory. The NetCDF file is generated using the module `save.py` from
-[`PyStemmusScope`](https://pystemmusscope.readthedocs.io/en/latest/autoapi/index.html).
+[`PyStemmusScope`](https://github.com/EcoExtreML/STEMMUS_SCOPE_Processing/blob/main/PyStemmusScope/save.py).
```

Comment:
```suggestion
[`PyStemmusScope`](https://pystemmusscope.readthedocs.io/en/v0.3.0/reference/#PyStemmusScope.save).
```

Reply:
It is usually better to point users to documentation rather than the source code. Here I suggested the documentation link to module `save`.